### PR TITLE
Add Fire Emblem Echoes: Shadows of Valentia

### PR DESCRIPTION
**Pull request by rubenlie**
Adds Fire Emblem Echoes: Shadows of Valentia.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Fire Emblem Echoes: Shadows of Valentia"
+description = "Fire Emblem Echoes: Shadows of Valentia is a tactical role-playing game developed by Intelligent Systems and published by Nintendo for the Nintendo 3DS handheld video game console in 2017."
```
Comment (Leo626) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: The release date is not needed so the `in 2017` part is not needed.

**Comment by rubenlie** [comment]
icon try2
new icon

**Comment by rubenlie** [comment]
I can't find a normal version of the box art that has the final ESRB rating.

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "00040000001B4000"
+region = "usa"
+release_date = "2017"
```
Comment (Leo626) [inline games/fire-emblem-echoes-shadows-of-valentia/game.dat]: Why did you change this?

**Comment by Leo626** [comment]
* The icon shouldn't be posted here. It should replace the icon in your [branch](https://github.com/rubenlie/citra-games-wiki/tree/pr-builder-fire-emblem-echoes-shadows-of-valentia-83/games/fire-emblem-echoes-shadows-of-valentia). Note that there is a version in it. You can uninstall the update to remove it and redo the icon.
* There was also a previous [PR](https://github.com/citra-emu/citra-games-wiki/pull/195/files) that wasn't finished so you can use the icon from that one which doesn't have the version number. There is also the eur release that you can use. Just add the jpn release as well.
* There is a high quality screenshot of the boxart [here](https://images-na.ssl-images-amazon.com/images/I/91f6zYaD9lL._AC_SL1500_.jpg). It's jpg so save it as png after resizing and compressing.
* See the example the example in the [contributing](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#metadata) guide for adding issues and releases for other regions ignoring testcases. Issues should be below the `description`.

File: games/fire-emblem-echoes-shadows-of-valentia/game.dat (hunk 2 above)
Comment (rubenlie) [reply games/fire-emblem-echoes-shadows-of-valentia/game.dat]: but didn't you say the in 2017 part was not needed?

**Comment by Leo626** [comment]
Superseded by #248
